sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -187,43 +187,14 @@ export class StreamingReceiver extends MessageReceiver {
     };
 
     this._onAmqpError = (context: EventContext) => {
-      const receiver = this.link || context.receiver!;
       const receiverError = context.receiver && context.receiver.error;
       if (receiverError) {
         const sbError = translate(receiverError) as MessagingError;
         logger.logError(sbError, `${this.logPrefix} An error occurred for Receiver`);
```

Comment:
I would recommend the format we use in the link close event here as well

```suggestion
        logger.logError(sbError, `${this.logPrefix} 'receiver_error' event occurred. The associated error is`);
```

Reply:
Same for the session case

Reply:
For a future PR, we can consider removing the handler for receiver_error and session_error event altogether because we believe that these events are always followed by the receiver_close and session_close events being called where we log the error anyway

Reply:
Done and yes, that'd be a cool change. Especially now that they're doing practically nothing.

Reply:
Filed as #12109